packages/webapp/src/_app/ui/action-button.tsx

```diff
@@ -24,12 +24,12 @@ export enum ActionButtonSize {
 }
 
 export enum ActionButtonType {
-    PRIMARY = "bg-blue-500 border-blue-500 text-white hover:bg-blue-600",
-    DISABLED = "border-gray-400 bg-gray-300 text-gray-500",
-    NEUTRAL = "bg-gray-50 text-gray-800 hover:bg-gray-200",
-    INDUCTION_STATUS_PROFILE = "bg-blue-500 border-blue-500 text-white hover:bg-blue-600",
-    INDUCTION_STATUS_CEREMONY = "bg-blue-500 border-blue-500 text-white hover:bg-blue-600",
-    INDUCTION_STATUS_ACTION = "bg-green-500 text-white hover:bg-green-600",
+    Primary = "bg-blue-500 border-blue-500 text-white hover:bg-blue-600",
+    Disabled = "border-gray-400 bg-gray-300 text-gray-500",
+    Neutral = "bg-gray-50 text-gray-800 hover:bg-gray-200",
+    InductionStatusProfile = "bg-blue-500 border-blue-500 text-white hover:bg-blue-600",
+    InductionStatusCeremony = "bg-blue-500 border-blue-500 text-white hover:bg-blue-600",
+    InductionStatusAction = "bg-green-500 text-white hover:bg-green-600",
```

Comment:
Since we are on it... It's a good time to consider getting rid of Enums for React props. The reason for that is that anytime we want to use the `ActionButton` we also need to import `ActionButtonSize` and `ActionButtonType` which kinda sucks... A solution for that with union types is: 

```tsx
// inside action-button.tsx 
type ActionButtonType =
    | "primary"
    | "disabled"
    | "neutral"
    | "InductionStatusProfile"
    | "InductionStatusCeremony"
    | "InductionStatusAction";
const ActionButtonTypes: { [key in ActionButtonType]: string } = {
    primary: "bg-blue-500 border-blue-500 text-white hover:bg-blue-600",
    disabled: "border-gray-400 bg-gray-300 text-gray-500",
    neutral: "bg-gray-50 text-gray-800 hover:bg-gray-200",
    InductionStatusProfile:
        "bg-blue-500 border-blue-500 text-white hover:bg-blue-600",
    InductionStatusCeremony:
        "bg-blue-500 border-blue-500 text-white hover:bg-blue-600",
    InductionStatusAction: "bg-green-500 text-white hover:bg-green-600",
};

// using action button externally:
<ActionButton type="primary" /> // reacty!
```

Reply:
I like it. Let's do it, but we'll have a separate PR soon for cleaning up our shared components (Text, buttons, etc.) I only changed this here since I was already working with enums and wanted consistency with the capitalization.